assets/js/src/newsletters/send.jsx

```diff
@@ -141,6 +141,12 @@ const NewsletterSend = React.createClass({
     ).done((response) => {
       // save template in recently sent category
       this.saveTemplate(newsletter, () => {
+        if (window.mailpoet_show_congratulate_after_first_newsletter) {
+          this.setState({ loading: false });
```

Comment:
I'm getting JS warning here.
```
mailpoet.js:4825 Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op. Please check the code for the Form component.
```

Reply:
It will probably happen also for activation.

Reply:
I will check it but are you sure the warning comes from here? The warning is talking about `Form` component.